pxr/usd/usdSkel/testenv/testUsdSkelValidators.cpp

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/pxr.h"
+#include "pxr/base/tf/token.h"
+#include "pxr/usd/sdr/registry.h"
+#include "pxr/usd/usd/validationRegistry.h"
+#include "pxr/usd/usd/validationError.h"
+#include "pxr/usd/usd/validator.h"
+#include <pxr/usd/usdSkel/validatorTokens.h>
+#include <pxr/usd/usdSkel/root.h>
+#include <pxr/usd/usdSkel/bindingAPI.h>
+#include <pxr/usd/usdGeom/mesh.h>
+#include <pxr/usd/usdGeom/primvarsAPI.h>
+
+#include <algorithm>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+void
+TestUsdSkelBindingAPIChecker()
+{
+    // Verify validator exists
+    UsdValidationRegistry &registry = UsdValidationRegistry::GetInstance();
+    const UsdValidator *validator = registry.GetOrLoadValidatorByName(
+            UsdSkelValidatorNameTokens->skelBindingApiAppliedChecker);
+    TF_AXIOM(validator);
+
+    // Create Stage and mesh with a skel binding property
+    UsdStageRefPtr usdStage = UsdStage::CreateNew("/Users/andrewbeers/test.usda");
+    UsdGeomMesh mesh = UsdGeomMesh::Define(usdStage, SdfPath("/SkelRoot/Mesh"));
+    UsdGeomPrimvarsAPI primvarsApi(mesh);
+    UsdGeomPrimvar jointIndicesPrimvar = primvarsApi.CreatePrimvar(TfToken("skel:jointIndices"), SdfValueTypeNames->IntArray, UsdGeomTokens->vertex);
+    jointIndicesPrimvar.Set(VtIntArray{0, 1, 2});
+
+    UsdValidationErrorVector errors = validator->Validate(mesh.GetPrim());
+
+    // test and verify error for not having a skel binding api applied
```

Comment:
Smallest of nitpicks: Comment style should be sentence case just to match the rest of the comments.